### PR TITLE
i18n(de): Add German translation

### PR DESCRIPTION
**Pull request by GrumiumXD**

### CODE REVIEW

File: i18n/de/cosmic_term.ftl [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+new-profile = Neues Profil
+make-default = Als Standard setzen
+
+## Settings
+settings = Einstellungen
+
+### Appearance
+appearance = Aussehen
+theme = Thema
+match-desktop = An system anpassen
```
Comment (leb-kuchen) [inline i18n/de/cosmic_term.ftl:33]: This is a noun

File: i18n/de/cosmic_term.ftl [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+find-previous = Vorheriges
+find-next = Nächstes
+
+# Menu
+
+## File
+file = Datei
+new-tab = Neuer Tab
+new-window = Neues Fenster
+profile = Profile
```
Comment (leb-kuchen) [inline i18n/de/cosmic_term.ftl:72]: `Profil`, this is a singular

File: i18n/de/cosmic_term.ftl [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+delete = Löschen
+import = Importieren
+import-errors = Importfehler
+
+## Profiles
+profiles = Profile
+name = Name
+command-line = Startbefehl
+tab-title = Überschrift
+tab-title-description = Standard Titel des Tabs überschreiben
```
Comment (leb-kuchen) [inline i18n/de/cosmic_term.ftl:22]: This should be one word. ´Standardtitel´

File: i18n/de/cosmic_term.ftl [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+edit = Bearbeiten
+copy = Kopieren
+paste = Einfügen
+select-all = Alles auswählen
+find = Suche
+
+## View
+view = Ansicht
+zoom-in = Schrift vergrößern
+zoom-reset = Standard Schriftgröße
```
Comment (leb-kuchen) [inline i18n/de/cosmic_term.ftl:87]: This should also be one world: `Standardschriftgröße`

**Review by leb-kuchen — CHANGES_REQUESTED** [review]

**Comment by GrumiumXD** [comment]
Yeah, shouldn't do things that rely on correct spelling in the middle of the night...

Thanks for checking. I corrected the areas you pointed out.

**Comment by leb-kuchen** [comment]
Looks fine

**Review by leb-kuchen — APPROVED** [review]

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**